go/vt/mysqlctl/backup.go

```diff
@@ -151,6 +151,7 @@ func Backup(ctx context.Context, params BackupParams) error {
 	if err != nil {
 		return vterrors.Wrap(err, "StartBackup failed")
 	}
+	params.Logger.Infof("Starting backup %v", bh.Name())
```

Comment:
We really want the backup process to print out the name of the backup. It's so useful. I'd also look into formalizing it into something that is machine readable rather than human readable.

Reply:
@shlomi-noach Would this have potential to create (a lot) of logging noise?